boards/arm/xilinx_kv260_starter_kit_r5/doc/index.rst

```diff
@@ -0,0 +1,79 @@
+.. _xilinx_kv260_starter_kit_r5:
+
+Xilinx KV260 Starter Kit RPU Cortex-R5
+######################################
+
+Overview
+********
+
+This configuration provides support for an ARM Cortex-R5 CPU and these devices:
+
+* ARM PL-390 Generic Interrupt Controller
+* Xilinx Zynq TTC (Cadence TTC)
+* Xilinx Zynq UART
+
```

Comment:
The R5 subsystem on Zynqmp is called the RPU, real-time processing unit.

It can operate as:
* two independent R5 cores with their own TCMs (tightly coupled memories)
* or as a single dual lock step unit with double the TCM size.

====
RPU is a term used in the code so it is good to define it.  The multiple modes of the RPU are pretty fundamental to understanding this platform so give an overview.

The above text is a suggestion.  Please feel free to reword an place at an appropriate spot.